local/src/core/ArgParser.cc

```diff
@@ -190,7 +190,7 @@ ArgParser::Command::Command(std::string const &name,
 // check if this is a valid option before adding
 void ArgParser::Command::check_option(std::string const &long_option,
                                       std::string const &short_option,
-                                      std::string const &key) const {
+                                      std::string const & /* key */) const {
```

Comment:
That's OK, but in C++ you can simply omit the parameter name.

Reply:
Yeah, I prefer the added semantics of having the commented out variable.